Getting-Started.adoc

```diff
@@ -87,15 +87,19 @@ These flags can be set when installing the chart or you can apply them during an
 Here is an example of upgrading the release and enabling the Metrics Dashboard:
 
 ----
-helm upgrade demo sk8s-incubator/sk8s --set enable.metricsDashboard=true
+helm upgrade riff riffrepo/riff --set enable.metricsDashboard=true
 ----
 
 See how to use the link:Monitoring.adoc#dashboards[dashboards] and run some link:samples/README.adoc[samples].
 
 ==== To tear it all down
 
 ----
-helm delete demo --purge
+helm delete riff --purge
 ----
 
+==== Installing Local Builds
+
+When building snapshot images locally, pass in a reference to the `values-snapshot.yaml` file, which contains snapshot tags:
 
+- `helm install riffrepo/riff --name=faas --values=config/values-snapshot.yaml`
```

Comment:
this should probably include `--set httpGateway.service.type=NodePort` or possible we should provide 2 separate sections here, one for minikube and one for external clusters

@trisberg WDYT?

Reply:
I'm preparing a commit with `values-snapshot.yaml` and `values-nodeport.yaml` that containes only those values that we want to override so we can install using:

```
helm install --name demo --values helm/values-snapshot.yaml,helm/values-nodeport.yaml riffrepo/riff
```

I'd prefer to put these files somewhere other than the `config` directory

Reply:
See: https://github.com/projectriff/riff/pull/186

Reply:
I've merged the PR for adding `values-snapshot.yaml` (with only versions) and `values-nodeport.yaml` into the `helm` subdirectory, so we need to adapt these instructions accordingly, and we can remove the `values-snapshot.yaml` included with this PR.